src/Tutorial.ts

```diff
@@ -141,7 +143,28 @@ export class Tutorial {
         return x
     }
 
+    lockEagleUi = () :void => {
+        $('div, button').css('pointer-events', 'none');
+        $("body").on('keydown.lockUi', function (event: JQuery.TriggeredEvent) {
+            event.preventDefault()
+            event.stopImmediatePropagation()
+            event.stopPropagation()
+        })
+    }
```

Comment:
there are a lot of elements being locked, however locking just parent divs for example, doesnt always apply to button children. we also need the entire ui to be locked, not just some select objects, because we dont want the ui to change at all while the tutorial system is reading it and showing the step